api/src/services/notifications.ts

```diff
@@ -16,33 +17,41 @@ export class NotificationsService extends ItemsService {
 	}
 
 	async createOne(data: Partial<Notification>, opts?: MutationOptions): Promise<PrimaryKey> {
+		const response = super.createOne(data, opts);
```

Comment:
These need to be awaited

```suggestion
		const response = await super.createOne(data, opts);
```